### PR TITLE
feature: adapter packages can be downloaded and configured

### LINKED ISSUE #5584: Surface downloadable adapters in the Package Manager
Adapters available for download should show up in the Package Manager. Adding one will add its corresponding schemas to the overall SDK and SDK-UI ones, allowing them to be seen by the Bot Settings page.

### LINKED ISSUE #5907: Adapters: Modal should have a link pointing to the adapter helplink in uischema if available.

### PR DESCRIPTION
**Pull request by beyackle**
## Description

This merges the Adapters feature branch into main, bringing with it the new section of bot project settings (under the "new creation experience") feature flag.

## Task Item

closes #5907
closes #5584

## Screenshots

image
image

### CODE REVIEW

File: Composer/packages/client/src/pages/botProject/BotProjectSettingsTableView.tsx [hunk 1]
```diff
@@ -40,12 +42,15 @@ export const BotProjectSettingsTableView: React.FC<RouteComponentProps<{
   const botProjects = useRecoilValue(localBotsDataSelector);
   const botProject = botProjects.find((b) => b.projectId === projectId);
   const isRootBot = !!botProject?.isRootBot;
+  const useAdapters = useFeatureFlag('NEW_CREATION_FLOW');
```
Comment (carlosscastro) [inline Composer/packages/client/src/pages/botProject/BotProjectSettingsTableView.tsx:45]: There is a constant for NEW_CREATION_FLOW in featureFlags.ts. The cool thing about that is that when this feature flag changes name or is deleted, this line would throw an error and we'd detect that we need to update the code.
Not sure if feature flags is accessible from here but worth considering.

Comment (beyackle) [reply Composer/packages/client/src/pages/botProject/BotProjectSettingsTableView.tsx:45]: Good catch. I'll fix that.

Comment (beyackle) [reply Composer/packages/client/src/pages/botProject/BotProjectSettingsTableView.tsx:45]: On second look, we don't need to. `useFeatureFlag` already takes a value of type `FeatureFlagKey` as an argument, so if that definition changes, this line will no longer typecheck, and that's exactly the error you mentioned.

File: Composer/packages/client/src/pages/botProject/styles.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+`;
+
+export const errorTextStyle = css`
+  margin-bottom: 5px;
+  font-size: ${FontSizes.small};
+`;
+
+export const errorIcon = {
+  root: {
+    color: '#A80000',
```
Comment (srinaath) [inline Composer/packages/client/src/pages/botProject/styles.ts:87]: if its an error icon color we need to use SharedColors or CommunicationColors

**Review by srinaath — DISMISSED** [review]

**Comment by cwhitten** [comment]
@carlosscastro let's go ahead and take this.. but can you address conflicts?

File: Composer/packages/client/src/pages/botProject/styles.ts (hunk 2 above)
Comment (beyackle) [reply Composer/packages/client/src/pages/botProject/styles.ts:87]: Nice catch. I'll fix that.

**Review by srinaath — DISMISSED** [review]

**Comment by carlosscastro** [comment]
To reviewers: a bit more code was added around package manager but it was reviewed and tested separately here: https://github.com/microsoft/BotFramework-Composer/pull/6035

**Review by srinaath — APPROVED** [review]

**Review by cwhitten — APPROVED** [review]

**Pull request merged**